geode-core/src/main/java/org/apache/geode/management/internal/cli/domain/IndexDetails.java

```diff
@@ -240,11 +250,20 @@ public String toString() {
     buffer.append(", regionPath = ").append(getRegionPath());
     buffer.append(", isValid = ").append(getIsValid());
     buffer.append(", projectAttributes = ").append(getProjectionAttributes());
+    buffer.append(", indexMaintStatus = ").append(getIndexMaintenance());
```

Comment:
It might be best to spell out the word maintenance?  We aren't saving a lot without it and the method itself already spells it out.